src/Pigeons.jl

```diff
@@ -63,7 +63,7 @@ const use_auto_exec_folder = ""
 
 include("includes.jl")
 
-export pigeons, Inputs, PT, 
+export pigeons, Inputs, PT, BreadCrumbs,
```

Comment:
Sounds good to export it, but then the PR should contain documentation (in code and in the website).